src/libs/fullstory/index.ts

```diff
@@ -27,12 +29,17 @@ const FS = {
      */
     onReady: () =>
         new Promise((resolve) => {
-            // Initialised via HEAD snippet
-            if (isInitialized()) {
-                init({orgId: ''}, resolve);
-            } else {
-                FullStory('observe', {type: 'start', callback: resolve});
-            }
+            Environment.getEnvironment().then((envName: string) => {
+                if (CONST.ENVIRONMENT.PRODUCTION !== envName) {
+                    return;
+                }
+                // Initialised via HEAD snippet
+                if (isInitialized()) {
+                    init({orgId: ''}, resolve);
```

Comment:
I am not sure if these two init functions are different, but it might be good to add some comment for why you need to `init` when its already `isInitialized`
```suggestion
                if (!isInitialized()) {
                    init({orgId: ''}, resolve);
```

Reply:
Done